apluslms_roman/cli.py

```diff
@@ -365,20 +416,136 @@ def build_action(context):
     return result.code
 
 
+def init_action(context):
+    cwd = getcwd()
+    name = ProjectConfig.DEFAULT_FILENAME
+    try:
+        # save will return false if the file already exists and the document hasn't been changed
+        config = get_config(context)
+        if not config.save(overwrite=False):
+            exit(1, _("A project configuration already exists at {}".format(config.path)))
+    # config file was not found in folder
+    except FileNotFoundError:
+        try:
+            ProjectConfig.create(path_join(cwd, name))
+        except ValidationError as e:
+            exit(1, '\n'.join(render_error(e)))
+    print(_("Project configuration file {} created successfully.".format(name)))
+
+
 def config_print_action(context):
     all_ = not any(getattr(context.args, k, False) for k in ('user', 'project'))
     if context.args.debug:
         print("---\n# arguments:")
         data = {k: v for k, v in vars(context.args).items() if k[0] != '_' and not callable(v)}
         yaml_dump(data, stdout)
-    if all_ or context.args.user:
+    if all_ or context.args.global_:
         print("---\n# roman settings:")
         yaml_dump(context.settings._data, stdout)
     if all_ or context.args.project:
         print("---\n# project config")
         yaml_dump(get_config(context)._data, stdout)
 
 
+def config_set_action(context):
+    if context.args.global_ == context.args.project:
+        raise Exception("Choose either global settings or project settings")
+    document = context.settings if context.args.global_ else get_config(context)
+
+    for val in context.args.values:
+        try:
+            key, val = val.split('=', 1)
+            document.mlset_cast(key, val)
+        except ValueError as err:
+            if hasattr(err, 'value_type'):
+                raise ValueError(_("{} should be of type '{}', but was '{}'.").format(
```

Comment:
We should avoid raising exceptions for the user. Basically, when the user doesn't benefit from the traceback, it will just fool them. Thus, prefer `exit(1, message)` here.